css/component_styles/grid.css

```diff
@@ -1,3 +1,52 @@
 @import url('./components.css');
 
-/* -------- grid css -------- */
\ No newline at end of file
+/* -------- grid css -------- */
+
+.grid-container {
+    display: grid;
+    background-color: var(--bg-color);
+    width: 100%;
+}
+
+.grid-item {
+    border: solid 2px var(--color-dnd);
+    padding: 1rem;
```

Comment:
It is suggested, that always use padding margin in multiples of 4 or 8 like padding you can give here 1.2 rem. It is a standard way of handling lengths

Reply:
Thanks for the suggestion I will follow this ✅